features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/AutoSessionScenario.m

```diff
@@ -0,0 +1,20 @@
+//
+// Created by Jamie Lynch on 07/06/2018.
+// Copyright (c) 2018 Bugsnag. All rights reserved.
+//
+
+#import "AutoSessionScenario.h"
+
+
+@implementation AutoSessionScenario
+
+- (void)startBugsnag {
+    self.config.shouldAutoCaptureSessions = YES;
+    [super startBugsnag];
+}
+
+- (void)run {
+    [self flushAllSessions];
```

Comment:
My main concern with this approach is that if we apply it to all the session scenarios, it's going to lead to very long CI build times, as it can take 60s for this to occur. I've updated the scenarios to send sessions naturally for now, but think we'll have to revisit this approach if it gets to the point where we have >5 session scenarios.

Reply:
Can you shorten the interval in tests? It sounds similar to something that I did in the tests [bugsnag-node](https://github.com/bugsnag/bugsnag-node/blob/1b6ba6dbd3ba18b4ee0f4a284318306a44bc5e66/lib/sessions/tracker.js#L11).

Reply:
Also did some casual time travelling https://github.com/bugsnag/bugsnag-node/blob/7f4ee1c1219216b9f19ab7ce5ea75c33e555388d/test/sessions/tracker.js#L12